docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -111,6 +111,112 @@ __Example__:
     - "10.0"
 ```
 
+## `devices`
+
+__Description__: Field for defining devices request.
+
+A real device can be selected using two different manner:
+- Specific device selection, through device IDs
+- Filtered device selection, with several contraints such as `name`, `platformVersion`, etc...
+
+When an ID is specified, it is prioritary on other settings.
+
+__Type__: *[]object*
+
+__Example__:
+```yaml
+devices:
+  - name: "Google Pixel.*"
+    platformVersion: 8.1
+  - id: Google_Pixel_2_real_us
+```
+
+### `id`
+
+__Description__: Request a specific device by its id.
+
+__Type__:  *string*
+
+__Example__:
+
+```yaml
+        id: Google_Pixel_2_real_us
+```
+
+### `name`
+
+__Description__:  Request a device by its name.
+
+__Type__:  *string*
+
+__Example__:
+
+Using complete name:
+```yaml
+      - name: Google Pixel 4 XL
+```
+
+Using pattern matching:
+```yaml
+        name: Google Pixel.*
+```
+
+### `platformVersion`
+
+__Description__: Request a device with a specific platform version.
```

Comment:
@sweeneyskirt-sl missed some comments.
`Request any device matching` let me though that it will be executed one per matching device.

Is it ok to replace any by `that the` ?
[Like here](https://github.com/saucelabs/sauce-docs/pull/404/files#r640006714)

Reply:
Definitely. I agree that is clearer. :)